ui/app/components/gas-customization/gas-modal-page-container/gas-modal-page-container.component.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import PageContainer from '../../page-container'
+import { Tabs, Tab } from '../../tabs'
+
+export default class GasModalPageContainer extends Component {
+  static contextTypes = {
+    t: PropTypes.func,
+  }
+
+  static propTypes = {
+    hideModal: PropTypes.func,
+  }
+
+  state = {}
+
+  renderBasicTabContent () {
+    return (
+      <div className="basic-tab"/>
+    )
+  }
+
+  renderAdvancedTabContent () {
+    return (
+      <div className="advanced-tab"/>
+    )
+  }
+
+  renderInfoRow (className, totalLabelKey, fiatTotal, cryptoTotal) {
+    return (
+      <div className={className}>
+        <div className="total-info">
+          <span className="total-label">{`${this.context.t(totalLabelKey)}:`}</span><span className="total-value">{fiatTotal}</span>
```

Comment:
Is `totalLabelKey` defined in `messages.json`? We probably want to omit 'Key' and 'Label' from from translation keys if possible.